src/Resolvers/Microsoft.DotNet.MSBuildSdkResolver/Strings.resx

```diff
@@ -130,7 +130,7 @@
     <value>Version {0} of the .NET SDK is smaller than the minimum version {1} required by Visual Studio. Check that a recent enough .NET SDK is installed or increase the version specified in global.json.</value>
   </data>
   <data name="UnableToLocateNETCoreSdk" xml:space="preserve">
-    <value>Unable to locate the .NET SDK. Check that it is installed and that the version specified in global.json (if any) matches the installed version.</value>
+    <value>Unable to locate the .NET SDK. Check that it is installed, your PATH is configured for the correct architecture, and that the version specified in global.json (if any) matches the installed version.</value>
```

Comment:
thoughts on an aka.ms link to https://github.com/dotnet/core/blob/main/release-notes/6.0/known-issues.md#2-upgrade-of-visual-studio-or-net-sdk-from-earlier-builds-can-result-in-a-bad-path-configuration-on-windows for follow up?  This is a good hint, but I fear that given the number of issues we've seen around it more direct links may be required.

Reply:
Chatted offline - that page isn't stable, so if we have a more permanent link I can come back and add it to this message. This is already a good change though, so let's take it.